src/lib/config.js

```diff
@@ -31,25 +33,36 @@ var getK8sClusterDomain = function() {
   var domain = process.env.KUBERNETES_CLUSTER_DOMAIN || "cluster.local";
   verifyCorrectnessOfDomain(domain);
   return domain;
+};
 
-  /**
-   * Calls a reverse DNS lookup to ensure that the given custom domain name matches the actual one.
-   * Raises a console warning if that is not the case.
-   * @param clusterDomain the domain to verify.
-   */
-  function verifyCorrectnessOfDomain(clusterDomain) {
-    var dns = require('dns');
-    if(clusterDomain && dns.getServers() && dns.getServers().length > 0) {
-      // In the case that we can resolve the DNS servers, we get the first and try to retrieve its host.
-      dns.reverse(dns.getServers()[0], function(err, host) {
-        if(err || host.length < 1 || !host[0].endsWith(clusterDomain)) {
-          console.warn("Possibly wrong cluster domain name! Detected '%s' but expected similar to: %s",  clusterDomain, host);
-        } else {
-          console.log("The cluster domain '%s' was successfully verified.", clusterDomain)
-        }
-      });
-    }
+/**
+ * Calls a reverse DNS lookup to ensure that the given custom domain name matches the actual one.
+ * Raises a console warning if that is not the case.
+ * @param clusterDomain the domain to verify.
+ */
+var verifyCorrectnessOfDomain = function(clusterDomain) {
+  if (!clusterDomain) {
```

Comment:
Prefer return early on negative if clauses vs nesting positive ones

Note: there's a couple of these that I've changed